hexrd/findorientations.py

```diff
@@ -57,35 +59,40 @@ def _process_omegas(omegaimageseries_dict):
 
 
 def label_spots(this_map, method, method_kwargs):
-    # !!! need to remove NaNs from map in case of eta gaps?
-    this_map[np.isnan(this_map)] = 0.
+    # !!! need to remove NaNs from map in case of eta gaps
+    # !!! doing offset and truncation by median value now
+    nan_mask = np.isnan(this_map)
+    med_val = np.median(this_map[~nan_mask])
+    this_map[nan_mask] = med_val
```

Comment:
Do you know if we ever confirmed that `this_map` here is a deep copy, so these modifications aren't getting pushed on to the indexing?

Reply:
I want the changes to persist to the indexing.